components/menu/menu.js

```diff
@@ -21,7 +20,7 @@ const keyCodes = {
  * @slot - Menu items
  * @fires d2l-menu-resize - Dispatched when size of menu changes (e.g., when nested menu of a different size is opened)
  */
-class Menu extends ThemeMixin(HierarchicalViewMixin(FocusVisiblePolyfillMixin(LitElement))) {
+class Menu extends ThemeMixin(HierarchicalViewMixin(LitElement)) {
```

Comment:
Comments on the original PR that added the polyfill here implied that this was needed solely to allow the polyfill to function on nested menu-items, so it seems like we also want it gone here.